src/main/fc/fc_core.c

```diff
@@ -513,6 +513,73 @@ void tryArm(void)
     }
 }
 
+static void handlePIDAntiWindup(throttleStatus_e throttleStatus)
+{
+    /* In airmode Iterm should be prevented to grow when Low thottle and Roll + Pitch Centered.
+       This is needed to prevent Iterm winding on the ground, but keep full stabilisation on 0 throttle while in air
+       Low Throttle + roll and Pitch centered is assuming the copter is on the ground. Done to prevent complex air/ground detections */
+
+    static bool antiWindupWasDeactivatedOnce = false;
+
+    // Track if ANTI_WINDUP was activated
+    if (!ARMING_FLAG(ARMED)) {
+        antiWindupWasDeactivatedOnce = false;
+    }
+    // In MANUAL mode we reset integrators prevent I-term wind-up (PID output is not used in MANUAL)
+    if (FLIGHT_MODE(MANUAL_MODE) || !ARMING_FLAG(ARMED)) {
+        DISABLE_STATE(ANTI_WINDUP);
+        pidResetErrorAccumulators();
+        return;
+    }
+
+    rollPitchStatus_e rollPitchStatus = calculateRollPitchCenterStatus();
+
+    // Set antiWindupWasDeactivatedOnce to prevent anti windup from being activated again
+    if ((throttleStatus != THROTTLE_LOW) && (rollPitchStatus != CENTERED)) {
+        antiWindupWasDeactivatedOnce = true;
+    }
+    // At non-zero throttle - always disable ANTI_WINDUP
+    if (throttleStatus != THROTTLE_LOW) {
+        DISABLE_STATE(ANTI_WINDUP);
+        return;
+    }
+    // This case applies only to MR when Airmode management is throttle threshold activated
+    if (rcControlsConfig()->airmodeHandlingType == THROTTLE_THRESHOLD) {
```

Comment:
This enabled THROTTLE_THRESHOLD to be active on Airplanes right now. Current code does not allow for such a behavior